ibm_catalog.json

```diff
@@ -130,7 +130,20 @@
               "key": "zone"
             },
             {
-              "key": "vpc_id"
+              "key": "has_existing_vpc",
+              "required": true              
+            },
+            {
+              "key": "vpc_id",
```

Comment:
This is not a review comment, just a note: I think we now have a way to provide a drop-down here and for subnets that has the existing IDs of that type in the user's account. I can look up how that is done, if that sounds like a good idea?